packages/nodes-base/nodes/HtmlTemplate/HtmlTemplate.node.ts

```diff
@@ -0,0 +1,140 @@
+import {
+	INodeExecutionData,
+	IExecuteFunctions,
+	INodeType,
+	INodeTypeDescription,
+} from 'n8n-workflow';
+
+export class HtmlTemplate implements INodeType {
+	description: INodeTypeDescription = {
+		displayName: 'HTML Template',
+		name: 'htmlTemplate',
+		icon: 'file:html.svg',
+		group: ['transform'],
+		version: 1,
+		subtitle: 'Render HTML',
+		description: 'Render an HTML template',
+		defaults: {
+			name: 'HTML Template',
+		},
+		inputs: ['main'],
+		outputs: ['main'],
+		parameterPane: 'wide',
+		properties: [
+			{
+				displayName: 'HTML',
+				name: 'html',
+				typeOptions: {
+					editor: 'htmlEditor',
+				},
+				type: 'string',
+				default: placeholder,
+				noDataExpression: true,
+				description: 'HTML template to render',
+			},
+			{
+				displayName:
+					'<b>Tips</b>: Type ctrl+space for completions. Use <code>{{ }}</code> for expressions and <code>&lt;style&gt;</code> tags for CSS. More options in the triple-dot button. JS in <code>&lt;script&gt;</code> tags is included but not executed.',
+				name: 'notice',
+				type: 'notice',
+				default: '',
+			},
+		],
+	};
+
+	async execute(this: IExecuteFunctions): Promise<INodeExecutionData[][]> {
+		const items = this.getInputData();
+		const returnData: INodeExecutionData[] = [];
+
+		for (let i = 0; i < items.length; i++) {
+			try {
+				let html = this.getNodeParameter('html', i) as string;
+
+				for (const resolvable of getResolvables(html)) {
+					html = html.replace(resolvable, this.evaluateExpression(resolvable, i) as any);
+				}
+
+				returnData.push(...makeEecutionData.call(this, html, i));
```

Comment:
since we already ship with handlebars, I suggest that we use that instead.

perhaps a bit like this:
```suggestion
				const html = this.getNodeParameter('html', i) as string;
				const ast = Handlebars.parse(html);
				const template = Handlebars.compile(ast);
				const payload: Record<string, any> = {};
				for (const statement of ast.body) {
					if (statement.type === 'MustacheStatement') {
						//@ts-ignore
						const key = statement.path.original;
						_.set(payload, key, this.evaluateExpression(`{{${key}}}`, i));
					}
				}
				returnData.push(...makeExecutionData.call(this, template(payload), i));
```

Reply:
Thanks, tried this but it fails to parse binary expressions. Reverted until I have more time since not familiar with this lib, will revisit before code review.

![image](https://user-images.githubusercontent.com/44588767/212015504-23907feb-99f8-4811-b763-3c5b0b791d9b.png)